### PR TITLE
chore(plugins): bump plugin version, also fix for offset_encoding

### LINKED ISSUE #2209: offset_encoding: expected string, got nil
### Problem description

Error autoimport-ing LSP suggestions

### LunarVim version

7125e2b chore: update changelog

### Neovim version (>= 0.6)

NVIM v0.7.0-dev+906-gde6f9233e Build type: Release LuaJIT 2.1.0-beta3 Compiled by xp42@deb

### Operating system/version

macOS 12.1 (21C52)

### Affected language servers

tsserver

### Steps to reproduce

_No response_

### Actual behavior

Begin by typing a component name, e.g. `<Butto|` 

LSP provides suggestion (see screenshots)

TAB + Enter of  autoimport suggestion produces the error.

### Expected behavior

_No response_

### log and support info

```shell
:LspInfo

`Language client log: /Users/xp42/.cache/nvim/lsp.log
~                 Detected filetype:   typescript
~
~                 2 client(s) attached to this buffer:
~
~                 Client: tailwindcss (id: 1, pid: 96286, bufnr: [5])
~                  filetypes:       aspnetcorerazor, astro, astro-markdown, blade, django-html, edge, eelixir, ejs, erb, eruby, gohtml, haml, handlebar
~                  autostart:       true
~                  root directory:  /Users/xp42/Dev/flowriter
~                  cmd:             tailwindcss-language-server --stdio
~
~                 Client: tsserver (id: 2, pid: 96287, bufnr: [5])
~                  filetypes:       javascript, javascriptreact, javascript.jsx, typescript, typescriptreact, typescript.tsx
~                  autostart:       true
~                  root directory:  /Users/xp42/Dev/flowriter
~                  cmd:             typescript-language-server --stdio
~
~                 Configured servers list: tailwindcss, tsserver`

`


:messages

`E5108: Error executing lua ...vim/HEAD-de6f923/share/nvim/runtime/lua/vim/lsp/util.lua:366: offset_encoding: expected string, got nil
stack traceback:
        [C]: in function 'error'
        vim/shared.lua: in function 'validate'
        ...vim/HEAD-de6f923/share/nvim/runtime/lua/vim/lsp/util.lua:366: in function 'apply_text_edits'
        ...unarvim/site/pack/packer/start/nvim-cmp/lua/cmp/core.lua:372: in function <...unarvim/site/pack/packer/start/nvim-cmp/lua/cmp/core.lua:344>
        ...te/pack/packer/start/nvim-cmp/lua/cmp/utils/feedkeys.lua:51: in function <...te/pack/packer/start/nvim-cmp/lua/cmp/utils/feedkeys.lua:49>
E481: No range allowed`



:checkhealth

`
null-ls: require("null-ls.health").check()
========================================================================
  - INFO: no sources registered

nvim: health#nvim#check
========================================================================
## Configuration
  - OK: no issues found

## Performance
  - OK: Build type: Release

## Remote Plugins
  - OK: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=^H
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
  - INFO: $TERM_PROGRAM='iTerm.app'
  - INFO: $COLORTERM='truecolor'

nvim-lsp-installer: require("nvim-lsp-installer.health").check()
========================================================================
## nvim-lsp-installer report
  - OK: neovim version >= 0.6.0
  - WARNING: **Composer**: not available
  - WARNING: **PHP**: not available
  - ERROR: **wget**: not available
  - OK: **tar**: `bsdtar 3.5.1 - libarchive 3.5.1 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.8 `
  - OK: **gzip**: `Apple gzip 352.40.6`
  - OK: **curl**: `curl 7.77.0 (x86_64-apple-darwin21.0) libcurl/7.77.0 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.42.0`
  - WARNING: **java**: not available
  - WARNING: **javac**: not available
  - OK: **python3**: `Python 3.9.10`
  - OK: **node**: `v14.16.0`
  - OK: **Ruby**: `ruby 2.6.8p205 (2021-07-07 revision 67951) [universal.arm64e-darwin21]`
  - OK: **Go**: `go version go1.17.6 darwin/amd64`
  - OK: **sh**: `Ok`
  - OK: **bash**: `GNU bash, version 3.2.57(1)-release (arm64-apple-darwin21)`
  - OK: **RubyGem**: `3.0.3.1`
  - OK: **npm**: `6.14.11`
  - OK: **pip3**: `pip 21.3.1 from /opt/homebrew/lib/python3.9/site-packages/pip (python 3.9)`

nvim-treesitter: require("nvim-treesitter.health").check()
========================================================================
## Installation
  - OK: `tree-sitter` found  0.20.1 (062421dece3315bd6f228ad6d468cba083d0a2d5) (parser generator, only needed for :TSInstallFromGrammar)
  - OK: `node` found v14.16.0 (only needed for :TSInstallFromGrammar)
  - OK: `git` executable found.
  - OK: `cc` executable found. Selected from { vim.NIL, "cc", "gcc", "clang", "cl", "zig" }
    Version: Apple clang version 13.0.0 (clang-1300.0.29.30)
  - OK: Neovim was compiled with tree-sitter runtime ABI version 13 (required >=13). Parsers must be compatible with runtime ABI.

## Parser/Features H L F I J
  - json           ✓ ✓ ✓ ✓ . 
  - c              ✓ ✓ ✓ ✓ ✓ 
  - css            ✓ . ✓ ✓ ✓ 
  - python         ✓ ✓ ✓ ✓ ✓ 
  - tsx            ✓ ✓ ✓ ✓ ✓ 
  - typescript     ✓ ✓ ✓ ✓ ✓ 
  - bash           ✓ ✓ ✓ . ✓ 
  - javascript     ✓ ✓ ✓ ✓ ✓ 
  - java           ✓ ✓ . ✓ ✓ 
  - lua            ✓ ✓ ✓ ✓ ✓ 
  - yaml           ✓ ✓ ✓ ✓ ✓ 
  - rust           ✓ ✓ ✓ ✓ ✓ 

  Legend: H[ighlight], L[ocals], F[olds], I[ndents], In[j]ections
         +) multiple parsers found, only one will be used
         x) errors found in the query, try to run :TSUpdate {lang}

provider: health#provider#check
========================================================================
## Clipboard (optional)
  - OK: Clipboard tool found: pbcopy

## Python 2 provider (optional)
  - WARNING: No Python executable found that can `import neovim`. Using the first available executable for diagnostics.
  - ERROR: Python provider error:
    - ADVICE:
      - provider/pythonx: Could not load Python 2:
          /usr/bin/python2 does not have the "neovim" module. :help |provider-python|
          /usr/bin/python2.7 does not have the "neovim" module. :help |provider-python|
          python2.6 not found in search path or not executable.
          /usr/bin/python does not have the "neovim" module. :help |provider-python|
  - INFO: Executable: Not found

## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Multiple python3 executables found.  Set `g:python3_host_prog` to avoid surprises.
  - INFO: Executable: /opt/homebrew/bin/python3
  - INFO: Other python executable: /usr/bin/python3
  - INFO: Python version: 3.9.10
  - INFO: pynvim version: 0.4.3
  - OK: Latest pynvim is installed.

## Python virtualenv
  - OK: no $VIRTUAL_ENV

## Ruby provider (optional)
  - INFO: Ruby: ruby 2.6.8p205 (2021-07-07 revision 67951) [universal.arm64e-darwin21]
  - WARNING: `neovim-ruby-host` not found.
    - ADVICE:
      - Run `gem install neovim` to ensure the neovim RubyGem is installed.
      - Run `gem environment` to ensure the gem bin directory is in $PATH.
      - If you are using rvm/rbenv/chruby, try "rehashing".
      - See :help |g:ruby_host_prog| for non-standard gem installations.

## Node.js provider (optional)
  - INFO: Node.js: v14.16.0
  - INFO: Nvim node.js host: /Users/xp42/.config/yarn/global//node_modules/neovim/bin/cli.js
  - OK: Latest "neovim" npm/yarn package is installed: 4.10.1

## Perl provider (optional)
  - ERROR: perl provider error:
    - ADVICE:
      - "Neovim::Ext" cpan module is not installed

telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 13.0.0
  - WARNING: fd: not found. Install [sharkdp/fd](https://github.com/sharkdp/fd) for extended capabilities

## ===== Installed extensions =====

## Telescope Extension: `fzf`
  - INFO: No healthcheck provided

## Telescope Extension: `notify`
  - INFO: No healthcheck provided

## Telescope Extension: `projects`
  - INFO: No healthcheck provided

vim.lsp: require("vim.lsp.health").check()
========================================================================
  - INFO: LSP log level : WARN
  - INFO: Log path: /Users/xp42/.cache/nvim/lsp.log
  - INFO: Log size: 1 KB

vim.treesitter: require("vim.treesitter.health").check()
========================================================================
  - INFO: Runtime ABI version : 13
  - OK: Loaded parser for bash: ABI version 13
  - OK: Loaded parser for c: ABI version 13
  - OK: Loaded parser for css: ABI version 13
  - OK: Loaded parser for java: ABI version 13
  - OK: Loaded parser for javascript: ABI version 13
  - OK: Loaded parser for json: ABI version 13
  - OK: Loaded parser for lua: ABI version 13
  - OK: Loaded parser for python: ABI version 13
  - OK: Loaded parser for rust: ABI version 13
  - OK: Loaded parser for tsx: ABI version 13
  - OK: Loaded parser for typescript: ABI version 13
  - OK: Loaded parser for yaml: ABI version 13

which_key: health#which_key#check
========================================================================
## WhichKey: checking conflicting keymaps
  - WARNING: conflicting keymap exists for mode **"n"**, lhs: **"gb"**
  - INFO: rhs: `<Cmd>lua require("Comment.api").call("toggle_blockwise_op")<CR>g@`
  - WARNING: conflicting keymap exists for mode **"n"**, lhs: **"gc"**
  - INFO: rhs: `<Cmd>lua require("Comment.api").call("toggle_linewise_op")<CR>g@`
`
```

### Screenshots

image

image

image

### PR DESCRIPTION
**Pull request by abzcoding**
# Description

Bump plugin version

Fixes #2209

### CODE REVIEW

**Comment by abzcoding** [comment]
@exponent42 can you check if this fixes your issue?

File: lua/lvim/plugins.lua [hunk 1]
```diff
@@ -8,21 +8,21 @@ local commit = {
...
+  nvim_cmp = "7cb14b0c0f926e3cd5ac4d2e19634f82534c9d3c",
   nvim_dap = "c9a58267524f560112ecb6faa36ab2b5bc2f78a3",
-  nvim_lsp_installer = "0f8e6a6557b1e4d685f5ad1bb511be11961e5969",
-  nvim_lspconfig = "58d2ba6b968539a20d701be0bf98ae154456e265",
+  nvim_lsp_installer = "cc34adb58de9e5a42af730c97ec04fc500bb5e63",
+  nvim_lspconfig = "ea29110765cb42e842dc8372c793a6173d89b0c4",
   nvim_notify = "15f52efacd169ea26b0f4070451d3ea53f98cd5a",
-  nvim_tree = "0a2f6b0b6ba558a88c77a6b262af647760e6eca8",
-  nvim_treesitter = "6039e4782a300b0570e54fc7fdcf3b4728203c77",
+  nvim_tree = "2dfed89af7724f9e71d2fdbe3cde791a93e9b9e0",
```
Comment (kylo252) [inline lua/lvim/plugins.lua:24]: This one is gonna be a headache.. 😢 
https://github.com/kyazdani42/nvim-tree.lua/commits/master

I say we leave it alone for now.

Comment (abzcoding) [reply lua/lvim/plugins.lua:24]: agreed

**Review by kylo252 — APPROVED** [review]

**Comment by kylo252** [comment]
I've merged those changed into master as well. Hopefully nothing is broken! 

@abzcoding, should we lock the requirement to v0.6.1+? Did you notice that it mattered?

**Comment by abzcoding** [comment]
yeah we should do that

**Pull request merged**